pkg/core/tekton/builder/pipeline_run.go

```diff
@@ -0,0 +1,104 @@
+package builder
+
+import (
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PipelineRunBuilder holds the tekton pipeline definition
+type PipelineRunBuilder struct {
+	PipelineRun v1beta1.PipelineRun
+}
+
+// NewPipelineRunBuilder creates a Pipeline with default values.
```

Comment:
It creates PipelineRun, not Pipeline, right?

Reply:
right